src/main/java/cc/polyfrost/oneconfig/gui/pages/Page.java

```diff
@@ -79,6 +82,12 @@ public void scrollWithDraw(long vg, int x, int y, InputHandler inputHandler) {
         ScissorHelper scissorHelper = ScissorHelper.INSTANCE;
         int maxScroll = getMaxScrollHeight();
         int scissorOffset = drawStatic(vg, x, y, inputHandler);
+        if (OneConfigGui.INSTANCE != null) {
+            if (!Objects.equals(previousSearch, OneConfigGui.INSTANCE.getSearchValue())) {
+                previousSearch = OneConfigGui.INSTANCE.getSearchValue();
+                finishUpAndClose();
```

Comment:
what exactly are we "finishing up and closing"... doesn't seem like best method name for this but idk

Reply:
legacy code